clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -635,23 +636,33 @@ public void testQuotaMetrics() throws Exception {
         assertEquals(300, maxMetric.value(), EPSILON);
     }
 
+    @Test
+    public void testGetOffsetsByTimes() {
```

Comment:
The multi-partition test is added in `PlaintextConsumerTest`. Would it be a little redundant to also test it here?